packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -459,16 +469,18 @@ export const browserTracingIntegration = ((_options: Partial<BrowserTracingOptio
               return;
             }
 
-            if (from !== to) {
-              startingUrl = undefined;
+            startingUrl = undefined;
```

Comment:
m/q: why did we remove the `from !== to` check?

Reply:
sorry, forgot to mention this - we actually have the same check in the history handler code, where we do not even trigger the handler in this case!

Reply:
nice, good catch!